script/test

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+#if [ $# -eq 0 ]; then
+#  echo "Linting code..."
+#  script/poetry run black .
+#fi
+
+echo "Running tests..."
+docker-compose run --service-ports --rm web bash -c "cd fixtures && ./copy_media.sh && cd .. && python manage.py test cms"
```

Comment:
I'm wondering if it isn't better to turn this around, so `script/test` executes the test suite, and if we want to run that from within docker we'd use `docker-compose run script/test`?